File: Modul-4/C/PRAK405-2310817320007-ZahraNabila.c
#include <stdio.h>

int main()
{
    int n, kelipatan;
    int hasil = 0;

    printf("Masukkan Jumlah Baris  = ");
    scanf("%d", &n);

    printf("Masukkan Kelipatan     = ");
    scanf("%d", &kelipatan);

    for (int i = 1; i <= n; i++) 
    {
        int TotalN = 0;

        for (int x = i; x > 0; x--)
        {

           TotalN = TotalN + x * kelipatan;
            if (x == i)
            {
                printf("(%d * %d)", x, kelipatan);
            }
            else
            {
                printf(" + (%d * %d)", x, kelipatan);
            }
        }
        printf(" = %d\n", TotalN);
        hasil = hasil + TotalN;
    }
    printf("Jumlah dari semua baris = %d\n", hasil);
    return 0;
}

File: Modul-4/Python/PRAK405-2310817320007-ZahraNabila.py
n = int(input("Masukkan Jumlah Baris  = "))
kelipatan = int(input("Masukkan Kelipatan     = "))
hasil = 0

for i in range(1, n + 1):
    TotalN = 0

    for x in range(i, 0, -1):
        TotalN = TotalN + x * kelipatan

        if x == i:
            print(f"({x} * {kelipatan})", end="")
        else:
            print(f" + ({x} * {kelipatan})", end="")

    print(f" = {TotalN}")
    hasil = hasil + TotalN

print(f"Jumlah dari semua baris = {hasil}")
